src/main/steps/c100-rebuild/hearing-without-notice/hearing-part2/content.ts

```diff
@@ -1,12 +1,13 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 import { YesOrNo } from '../../../../app/case/definition';
 import { TranslationFn } from '../../../../app/controller/GetController';
 import { FormContent } from '../../../../app/form/Form';
 import { isFieldFilledIn } from '../../../../app/form/validation';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-const en = () => ({
-  title: 'Details of without notice hearing',
-  line1: 'Give details of why you’re asking for a without notice hearing',
+export const en = () => ({
+  hearing2title: 'Details of without notice hearing',
```

Comment:
@DaimDN Please do not change the title field name for any screen. This is used by Google analytics, we should have title field for each page.
Can we revert it back ?